backend/src/main/kotlin/cz/loono/backend/db/model/HealthcareProvider.kt

```diff
@@ -0,0 +1,156 @@
+package cz.loono.backend.db.model
+
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.IdClass
+import javax.persistence.JoinColumn
+import javax.persistence.JoinTable
+import javax.persistence.ManyToMany
+import javax.persistence.Table
+
+@Entity
+@IdClass(HealthcareProviderId::class)
+@Table(name = "\"healthcare_provider\"")
+data class HealthcareProvider(
+
+    @Id
+    @Column(name = "location_id", nullable = false, columnDefinition = "TEXT")
+    val locationId: String = "",
+
+    @Id
+    @Column(name = "institution_id", nullable = false, columnDefinition = "TEXT")
+    val institutionId: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val code: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val title: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val institutionType: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val city: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val postalCode: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val street: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val houseNumber: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val region: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val regionCode: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val district: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val districtCode: String = "",
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val administrativeDistrict: String = "",
```

Comment:
Just to be sure - All of these columns are GUARANTEED to be present in the CSV, yes? If not, then let's use `nullable = true` with default `null` for those columns which might not be filled in.

What do you think?

Reply:
YES, there are guaranteed. the columns are strictly contained in the file in the same order. The column can be empty then you can find just separators there like `something,,,something else`.  We can discuss if instead of empty strings allow null values, how we interpret "no value" in the CSV file.

Reply:
Yep, I was asking about missing column values, not missing columns. So the actual values are not guaranteed and can be missing. IMO the best approach is to make those optional columns nullable..